project_internal_access_from_portal/tests/test_portal_internal_access.py

```
@@ -0,0 +1,72 @@
+            self.task.with_user(self.portal).write({"name": "X"})
+
+    def test_task_create_forbidden(self):
+        """Portal user cannot create task even after subscribing"""
+        self.project_pigs.message_subscribe([self.portal.partner_id.id])
+        self.task.flush_model()
+        with self.assertRaises(AccessError):
+            self.env["project.task"].with_user(self.portal).create(
+                {
+                    "name": "ShouldFail",
+                    "project_id": self.project_pigs.id,
+                }
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)